pymc3/tests/test_distributions.py

```diff
@@ -284,6 +284,10 @@ def mvt_logpdf(value, nu, Sigma, mu=0):
     return log_pdf
 
 
+def AR1_logpdf(value, k, tau_e):
+    return (sp.norm(loc=0,scale=1/np.sqrt(tau_e)).logpdf(value[0]) +
```

Comment:
- Sorry about the test.  I didn't know the best way to write one without introducing additional dependencies.  
- As it written now, the AR(1) likelihood calculation is conditional on $x_0 = 0$, which seems a bit arbitrary to me.  It seems more natural to conditional on the initial observation.  
- You could use the unconditional distribution to calculate the likelihood of the initial observation (as in stats models), but this only makes sense if the process is stationary ($ |k| < 1 $).  Maybe something like `init` in `GaussianRandomWalk`?
- I am trying to write a more general AR RV, but I'm getting a bit tripped up by creating the matrix of lagged observations in theano.